hppc/src/main/java/com/carrotsearch/hppc/Accountable.java

```diff
@@ -0,0 +1,21 @@
+package com.carrotsearch.hppc;
+
+/**
+ * Anything that could be accounted for memory usage
+ *
+ * Partly forked from Lucene
+ * tag releases/lucene-solr/8.5.1
+ */
+public interface Accountable {
+    /**
+     * Memory usage estimation
+     * @return Ram used in bytes
+     */
+    public long ramUsageBytes();
```

Comment:
I would rename this to "ramBytesAllocated" and change occupancyRate to "ramBytesUsed", returning the number of bytes actually in use. This makes it easy to compute the fraction of memory used and is more exact/ useful than a float?

Reply:
Yeah that is a good idea. Available in new commit